app/components/features/target-spf-item/component.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Story } from '@storybook/react/types-6-0';
+import TargetSPFItem from './component';
+import { TargetSPF, TargetSPFItemProps, Type } from './types';
+
+export default {
+  title: 'Components/Features/TargetSPF-Item',
+  component: TargetSPFItem,
+  argTypes: {},
+};
+
+const Template: Story<TargetSPFItemProps> = ({ ...args }: TargetSPFItemProps) => (
+  <TargetSPFItem {...args} />
+);
+
+export const Default = Template.bind({});
+Default.args = {
+  targetSPF: {
+    id: 1,
+    isAllTargets: false,
+    targetValue: 0.17,
+    spfValue: 1,
+    type: Type.BIOREGIONAL,
+    surface: '30 km²',
+    name: 'Inland water',
+  },
+  onRemove: (value: TargetSPF) => console.info('Remove: ', value),
+  onChange: (value: TargetSPF) => console.info('Change', value),
```

Comment:
You defined onChange only for the SPF value. 
We need to define 2 `onChangeSPF` and `onChangeTarget`

Reply:
In reality the function `onChange` is triggered both when `spf` and `target` are changed - either by changing the text input value or the slider. The value passed as a parameter is an object of type `TargetSPF` having all its fields up to date.

Reply:
@mbarrenechea ☝️